ManPac/Assets/Scripts/Player/Death/DeathHandler.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class DeathHandler : MonoBehaviour
+{
+
+    [SerializeField] private Animator GhostAnimator;
+    [SerializeField] private Transform HomePosition;
+
+    private PlayerInput _thisPlayerInput;
+    private PlayerMovement _thisPlayerMovement;
+    private IntersectionTraverser _thisPlayerTraverser;
+    //private bool _isActivePlayer = true;
+    private bool _isInDanger;
+    private bool _canDie;
+
+    public bool IsInDanger { private get => _isInDanger; set => _isInDanger = value; }
+    public bool CanDie { get => _canDie; set => _canDie = value; }
+
+    private void Start()
+    {
+        _canDie = true;
+        _isInDanger = false;
+        _thisPlayerTraverser = GetComponent<IntersectionTraverser>();
+        _thisPlayerMovement = GetComponent<PlayerMovement>();
+        _thisPlayerInput = GetComponent<PlayerInput>();
+        
+        _thisPlayerInput.enabled = true;
+        
+    }
+
+    public void CallDeath()
+    {
+        if (!CanDie)
+            return;
+
+        StartCoroutine(Death());
+    }
+    private IEnumerator Death()
```

Comment:
can you add a whitespace here
```suggestion
    }
    
    private IEnumerator Death()
```